### ISSUE TITLE
[Proposal] Change the "pink" Can you help us improve the documentation links

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#6505]
## Proposal

It is proposed, that we change the links as shown in the screenshot below. 

 - At the moment the "Find out how" points to valid but old videos. ... 
   - The GitHub Ui has changed quite a bit since then. I could use the new videos that I made lately or create new ones if needed. 
 - The "edit this tiddler on GitHub" should be changed and point to eg: 
   - https://saqimtiaz.github.io/tw5-docs-pr-maker/#HelloThere
   - It would be nice if we could change the behaviour at this page a bit. 
     - So **Make a PR** and **HelloThere** should be opened.
     - Make a PR should start with the text: **If you are new here read the [[Notes]] first**, So users know what to do.
     - I could create the needed PRs
  - We could add a new link to the Notes tiddler
     - Find out more about the [7 Steps to improve the TiddlyWiki documentation](https://talk.tiddlywiki.org/t/7-steps-to-improve-the-tiddlywiki-documentation/2590)

![grafik](https://user-images.githubusercontent.com/374655/156742755-31e61448-def9-4ec4-bcc6-f8f9cd735a60.png)

**Comment by Jermolene** [comment]
Hi @pmario I'd welcome improvements here.

I've always felt that linking directly to GitHub was a bit jarring. Perhaps clicking the ribbon should popup a modal with better instructions:

> Can you help improve "name of tiddler"?

> TiddlyWiki's documentation is written collaboratively by people like you contributing improvements. An individual contribution can be as small as adding a missing comma, but gradually they build up to a useful body of work.

> 1. Use [[PR maker]] to suggest an improvement
> 2. The suggestion is reviewed by the core team and then "merged" to become part of https://tiddlywiki.com

> No technical knowledge is required to complete the process.

> [[Click here to start]]

> If you know GitHub and Pull Requests you can use this link to edit this tiddler directly: [[Edit "name of tiddler" on GitHub]]

**Comment by pmario** [comment]
I think as soon as we mention GitHub or Pull Request many of our users start to build up a "mental wall", which is understandable, but shouldn't be the case.

I think we may rename GitHub with "tiddlywiki-com" or may be **"tiddlywiki-docs"**  ... Even changing the name of the branch here at GH, if possible. 

Other "techy-terms" to be renamed:

 - "create a Pull Request" with **"initiate a contribution"** or **"create a contribution"**
 - an "open PR" could be ... **"contribution under consideration"** or **considered contribution**
 - "merged pull request" with **"activated contribution"**

just some thoughts.

**Comment by Jermolene** [comment]
Won't most users eventually have to visit and/or understand GitHub so that they can trace what happened to their request?

**Comment by pmario** [comment]
Yes. ... But I do have the feeling, that many users even don't even open https://saqimtiaz.github.io/tw5-docs-pr-maker/ because we "advertise it" in the wrong way. ... 

I was thinking about the possibility to skip the CLA signing for documentation in favour of the [Developer Certificate of Origin](http://developercertificate.org/) "Sign off" mechanism. .... 

Which imo also has some disadvantages. From my point of view, it needs a valid e-mail address _and_ it has to be done with every single commit. ... but

we may have the possibility to create a workflow, that makes it sure, that the users are aware about the licenses used. It would make the 1-click experience more complicated. 

They still would need to touch github for the access token ... hmmmm there is always a but :/

**Comment by saqimtiaz** [comment]
> Yes. ... But I do have the feeling, that many users even don't even open https://saqimtiaz.github.io/tw5-docs-pr-maker/ because we "advertise it" in the wrong way. ...

Keep in mind that what is currently there is a proof of concept and we have never really taken the step to spread the word about it at all until your recent posts. This is reflected in the nature of the notes in the edition as well.



> I was thinking about the possibility to skip the CLA signing for documentation in favour of the [Developer Certificate of Origin](http://developercertificate.org/) "Sign off" mechanism. ....

It would probably be possible to integrate the CLA signing into the PR if that was desirable. However, I don't think we should put further work into any such improvements until and unless we see that the CLA step is the hurdle that stops people.

 The number of people that have actually tried to create a PR via that wiki is less than 5 and that includes everyone in this thread.

**Comment by pmario** [comment]
>The number of people that have actually tried to create a PR via that wiki is less than 5 and that includes everyone in this thread. 

Yes. I think we should try to change that. 

From all the conversations about documentation improvements at TTalk I see, that as soon as we start to write about GitHub and pull requests, the responses between the lines are: "This is for developers, for me as an ordinary user ... "

I think we need to make it clearer, that creating "contributions" using TiddlyWiki itself can be a "game changer". 

I will use it myself, because it's less work than creating a PR with my local development environment. ... Sure, there can always be improvements, but the "proof of concept" from my point of view is pretty solid already. 

The only important thing is to make backups, before you reload the page :)

**Comment by Marxsal** [comment]
Another improvement is to allow people to start tiddler discussions directly on Discourse:
![2022-03-07-screenshot](https://user-images.githubusercontent.com/8988994/157113085-b387c1a2-3713-4632-81e4-a66a9cce4e25.png)

After discourse on Discourse, the changes can be submitted by volunteers as a PR, so people don't have to actually deal with GitHub.
